index.js

```diff
@@ -119,7 +103,46 @@
   //  $$type :: String
   var $$type = '@@type';
 
-  //  type :: Any -> String
+  //  RPARSE :: RegExp
+  var RPARSE = /^([^]+)\/([^]+?)(?:@(\d+))?$/;
```

Comment:
I'm still very much in favour of the version with comments I proposed in [#8 (comment)][1].


[1]: https://github.com/sanctuary-js/sanctuary-type-identifiers/pull/8/files#r112677466